resources/assets/helpers/data.js

```diff
@@ -0,0 +1,200 @@
+/* global Date, JSON, Math, Object */
+
+import {
+  isArray,
+  isEmpty,
+  isNil,
+  isNull,
+  isObjectLike,
+  isUndefined,
+  mapValues,
+  omitBy,
+} from 'lodash';
+
+/**
+ * Convert an int to a string. (Supports 0-10)
+ * @param  {int} number
```

Comment:
```suggestion
 * @param  {Number} number
```
Is this intentionally an `{int}`?

Reply:
I just copy-🍝 the code, but good catch! Will update.